packages/data-point/lib/entity-types/entity-collection/resolve.test.js

```diff
@@ -94,14 +94,14 @@ describe('entity.collection.filter', () => {
     })
   })
 
-  test('should resolve filter Transform', () => {
+  test('should resolve filter Transform | values of collection 3 passed into acc', () => {
     return transform('collection:c.2', testData).then(acc => {
       expect(acc.value).toEqual([1, 3])
     })
   })
 
   // NOTE: to skip map execution when filter is empty
-  test('should skip Filter Transform if empty', () => {
+  test('should skip Filter Transform if empty | values fo collection 3 passed into acc', () => {
```

Comment:
Can we add `It` to the beginning of this description? I don't think we need the stuff from the `|` on.